src/languages/es-ES/commands/gameIntegration.json

```diff
@@ -144,10 +144,12 @@
 			]
 		],
 		"examples": [
+			"item poción",
```

Comment:
Does FFXIV understand Spanish names, tho?

Reply:
You're the one who added this lmfao